bBridgeAPISDK.Test/TestAuthorization.cs

```diff
@@ -11,15 +12,18 @@ public class TestAuthorization
         [Fact]
         public void TestCanReceiveAuthorizationTokenForUserNameAndPassword()
         {
-            var baseUri = new Uri(Resources.bBridgeAPIBaseURI);
-
             IAuthorizer userPasswordAuthorizer = new LazyCredentialsAuthorizer(
```

Comment:
You use same code to create `LazyCredentialsAuthorizer` at least 4 times. Maybe move it to some common method?